galaxy/templates/deployment-web.yaml

```diff
@@ -75,6 +91,15 @@ spec:
             periodSeconds: 60
             failureThreshold: 10
           volumeMounts:
+            {{- range $key, $entry := .Values.initScripts -}}
```

Comment:
This section is duplicated between web and job handlers. Should we consider moving the whole thing to a template to avoid repetition? (In fact, we have a fair bit of duplicate logic between the two, and I'm just raising this here as a general design issue)

Reply:
Yes, there are many many duplicates, with only changes being `applyToJob` vs `applyToWeb` as the relevant boolean. One concern I have about adding more things to the helpers is readability of the templates.